Dockerfile

```diff
@@ -324,6 +325,16 @@ RUN pkg \
         /usr/local/lib/node_modules/ts-protoc-gen
 RUN install -D protoc-gen-ts /out/usr/bin/protoc-gen-ts
 
+FROM sbtscala/scala-sbt:${SCALA_SBT_VERSION} as protoc_gen_scala
+RUN mkdir -p /scala-protobuf
+ARG PROTOC_GEN_SCALA_VERSION
+RUN curl -sSL https://api.github.com/repos/scalapb/ScalaPB/tarball/${PROTOC_GEN_SCALA_VERSION} | tar xz --strip 1 -C /scala-protobuf
+WORKDIR /scala-protobuf
+RUN gu install native-image
+ENV NATIVE_IMAGE_INSTALLED=true
```

Comment:
Is this only relevant at build time or do we also need to include it at runtime?